websites/K/Kinopub/metadata.json

```
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "service": "Kinopub",
+  "author": {
+    "id": "257151906038677504",
+    "name": "SHISEN#8023"
+  },
+  "description": {
+    "en": "Website for online watching movies, series, sports broadcasts and concerts",
```

I'd also mention the podcast part (cuz the official thing mentions it)
```suggestion
    "en": "Website for listening to podcasts and watching movies, series, sports broadcasts and concerts all online",
```

---

> I'd also mention the podcast part cuz the official thing mentions it. (Also changed the Russian to match the English one.

Website does not have a listening to podcasts feature.

---

ok